Health-and-wellness-mobile/src/views/login_view.scss

```diff
@@ -0,0 +1,21 @@
+.content {
+    align-content: center;
+}
+.image {
+    display: block;
```

Comment:
I don't think this is necessary, because the display characteristic applies the rule to components inside of this component